README.md

```diff
@@ -6,79 +6,64 @@ transfers made easy<br><br>
 </h3>
 
 
-
 [![CI](https://github.com/astronomer/apache-airflow-provider-transfers/actions/workflows/ci-uto.yaml/badge.svg)](https://github.com/astronomer/apache-airflow-provider-transfers)
 
-The **Universal Transfer Operator** allows data transfers between any supported source and target Datasets in [Apache Airflow](https://airflow.apache.org/). It offers a consistent agnostic interface, simplifying the users' experience, so they do not need to use specific providers or operators for transfers. The Universal Transfer Operator is maintained by [Astronomer](https://astronomer.io).
+The **Universal Transfer Operator** simplifies how users transfer data from a source to a destination using [Apache Airflow](https://airflow.apache.org/). It offers a consistent agnostic interface, simplifying the users' experience, so they do not need specific providers or operators. It is maintained by [Astronomer](https://astronomer.io).
```

Comment:
```suggestion
The **Universal Transfer Operator** simplifies how users transfer data from a source to a destination using [Apache Airflow](https://airflow.apache.org/). It offers a consistent agnostic interface, simplifying the users' experience, so they do not need specific operators. It is maintained by [Astronomer](https://astronomer.io).
```

Reply:
I think we should remove providers since we do need to install them as a dependency. WDYT?

Reply:
Users do not have to use them from a DAG authoring perspective - so I think it is worth keeping - but we could improve the phrasing, do you have any ideas?